client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -263,14 +256,11 @@ module.exports = {
 
 		renderPage(
 			<DomainManagementData
+				component={ DomainManagement.AddGoogleApps }
 				context={ context }
 				productsList={ productsList }
-				sites={ sites }>
-				<DomainManagement.AddGoogleApps
-					productsList={ productsList }
```

Comment:
`productsList` is now no longer passed to `AddGoogleApps` - but it seems that it does not need it. So all good, but it got me worried for a while